### PR TITLE
replace the button color to base color

### LINKED ISSUE #196: Pokemon card button font color
## Expected Behavior

Contrast the color for better visibility

## Actual/Current Behavior

Color contrast is not enough

## Steps to Reproduce the Problem

  1. Hover over the pokemon cards

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Font in the card buttons changed to base color

## Purpose
More contrast

## Approach
SCSS

Closes #196

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Easy change. Nice code

**Review by BlackCubes — APPROVED** [review]
It is early. GOOD JOB!!!!!!!!!!!!!!!!!!

File: pokeapp/src/assets/uiIcons/pokeDollar.svg [hunk 1]
```diff
@@ -6,7 +6,7 @@
  preserveAspectRatio="xMidYMid meet">
 
 <g transform="translate(0.000000,1644.000000) scale(0.100000,-0.100000)"
-fill="#fff" stroke="none">
+fill="hsl(0, 0%, 11%)" stroke="none">
```
Comment (kmausolf) [inline pokeapp/src/assets/uiIcons/pokeDollar.svg:9]: Ah nice, a grey instead of black?  Interested to see how this looks!

**Review by dannysanchez559 — APPROVED** [review]

File: pokeapp/src/assets/uiIcons/pokeDollar.svg (hunk 1 above)
Comment (gonzalespaulb) [reply pokeapp/src/assets/uiIcons/pokeDollar.svg:9]: dark dark grey yes. more contrast

**Pull request merged**